### ISSUE TITLE
Request a Live Demo

### ISSUE THREAD

**Issue by lidoma** [sehmaschine/django-grappelli#266]
Hi.
Is there a live demo of this admin interface?

**Comment by nerogit** [comment]
+1

**Comment by derek-adair** [comment]
I would also like to see a demo that showcases all of the features grappelli has to offer.  Its not exactly clear what I am getting, particularly when i install it on top of a fresh django instance (how i typically test out these types of things).

Seems like it would do a lot to advertise for your project!

**Comment by sehmaschine** [comment]
we decided that this is out of scope. besides, there's no need (at least not for us) to advertise the project.

**Comment by derek-adair** [comment]
Fair enough, just some feedback from a... kindof user.  I'm actually trying to find out what features you guys have added since[ mezzanine forked](https://github.com/stephenmcd/grappelli-safe).  Its difficult for me to compare without having them both at my fingertips.  I can't really spend the time to research the benefits from upgrading at this point and time...

I'd absolutely love to be able to compare features so as I can come up with some incentive for the community to embrace the idea of upgrading.  Seems like an obvious win in terms of maintenance, but i'm still getting my feet wet in mezzanine.

thoughts?

**Comment by sehmaschine** [comment]
I see your point ... but while you don't have the time to research the benefits, we don't have the time to setup and maintain a demo site (that said, I guess researching the benefits is less time-consuming).

**Comment by derek-adair** [comment]
Hmm, yes I can see how if you dont need or have any desire for driving people to this project a live demo is just simply not worth the time or effort.

What about some kind of support script to install a feature demonstration locally?

Mezzanine has a cool [createdb management command](https://github.com/stephenmcd/mezzanine/blob/280642c8d0e7e41b90398c974a15533f6df20049/mezzanine/core/management/commands/createdb.py) that allows users to explore the more advanced features of the project.  This would essentially amount to configuring the "setup" of the demo.

If this is something you find interesting I can keep this in mind when I do have time to explore this project more.  Thanks for your prompt responses!

**Comment by sehmaschine** [comment]
Not sure about that one since it's fairly easy to use grappelli with an existing project. So you just need to setup a project and install grappelli.

However, you could use the model available with grappelli/tests in order to work on this feature. If you send a PR I'm happy to review.

**Comment by doganmeh** [comment]
A live demo for open source projects is not common practice, although it would be great if there was one. 

But I do find grappelli's documentation not self-explanatory despite being well written. It lacks pictures, as a picture may worth a thousand word sometimes. To be able to follow along grappelli's documentation, you must have mastered how to customize django admin beforehand, read carefully, and process everything in your head to figure out what should it be doing or what should be the end result. That hurts.. and most people are not django admin guru. 

For example because grappelli described itself as such: "Grappelli adds a consistent and grid-based look & feel and some nifty features." I spent a considerable amount of time at the documentation trying to figure out how to do grids with grappelli, and it turned out to be (after hours of reading) a standard django feature.

Bottom line, authors may not need advertising their project, but the documentation is there for a reason, and I think it could be more accessible. 

Appreciate your work.

**Comment by sehmaschine** [comment]
@doganmeh a grid-based look & feel is definitely not a standard feature (django doesn't care at all about the UI).

**Comment by doganmeh** [comment]
Here is the django feature that I was referring to:

https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display

Briefly; when field names list such as:

    list_display = ('first_name', 'last_name')

fields of the model/object can be shown on separate columns, instead of the unicode representation of the object.

**Comment by parsch** [comment]
@doganmeh Grappelli is based on a very different grid system, on which allows controlling horizontal and vertical spacings throughout the whole design process, see http://muellergridsystem.com/

**Comment by doganmeh** [comment]
Now I see where you are coming from :) Thanks,
